webapp/templates/anomalies.html

```diff
@@ -1,11 +1,14 @@
 <!DOCTYPE html>
 <html>
-{{ template "_head.html" }}
+<head>
+  {{ template "_head_common.html" }}
+  <link rel="import" href="/components/anomalies.html">
+</head>
 <body>
 <div id="content">
   {{ template "_header.html" }}
   <div>
-    Interoperability anomalies view coming soon.
+    <wpt-anomalies pass-rate-metadata="{{ .Metadata }}" browser="{{ .Browser }}"></wpt-anomalies>
```

Comment:
When I view source on the link you provided, I'm seeing `browser=""` (but plenty of `pass-rate-metadata="..."`). Is that expected?

Reply:
If you set ?browser=chrome, it will set the browser-specific view up.

Reply:
> If you set ?browser=chrome, it will set the browser-specific view up.

When submitting PRs for review, please include as much information as possible. Thanks.